### ISSUE TITLE
[VB6] Failed to parse the class name

### ISSUE THREAD

**Issue by chenzimin** [antlr/grammars-v4#3967]
I tried to parse the `vb6/examples/form1.vb` file and got the same error in `vb6/examples/form1.vb.errors`, which is:

```
line 1:13 mismatched input 'Form1' expecting <EOF>
``` 

Anyway we can fix this?

**Comment by kaby76** [comment]
VB6 does not have classes.

**Comment by chenzimin** [comment]
You are right, I was confused between VB6 and VB.NET. I wrongly thought that `vb6/examples/form1.vb` should be correctly parsed by VB6.

Do you have if we have a grammar for VB.NET?

**Comment by kaby76** [comment]
> Do you know that if ANTLR4 have a grammar for VB.NET?

Pretty sure we don't have one. It should be added to the "to do" list.

If you can, rename the title of the issue to something like "New grammar request: VB.NET".

**Comment by Beakerboy** [comment]
I have a mostly complete grammar for VBA 7.1, which might to close to .Net
https://github.com/Beakerboy/grammars-v4/tree/coverage/vba/vba7_1

**Comment by chenzimin** [comment]
I tried to parse `vb6/examples/form1.vb` with https://github.com/Beakerboy/grammars-v4/tree/coverage/vba/vba7_1, it failed with a different error message:

```
line 1:0 extraneous input 'Public' expecting {'ATTRIBUTE', 'VERSION', NEWLINE, REMCOMMENT, COMMENT, WS}
line 1:7 no viable alternative at input ' Class'
```

https://pypi.org/project/antlr4-vba/ seems to use the same vba grammar as in this repository, and parsing `vb6/examples/form1.vb` file with vba.g4 will also return `line 1:13 mismatched input 'Form1' expecting <EOF>`

**Comment by Beakerboy** [comment]
VBA files have a required module header similar to:

```
Attribute VB_Name = “Foo”
```

since the example is missing that, it will not parse.

**Comment by chenzimin** [comment]
Thanks for the tip, but adding the header `Attribute VB_Name = "Foo"` to `vb6/examples/form1.vb` still results in `line 3:13 mismatched input 'Form1' expecting <EOF>`.

**Comment by Beakerboy** [comment]
Is the file missing a newline character at the end?
